pandas/core/generic.py

```diff
@@ -138,6 +138,9 @@ def _init_mgr(self, mgr, axes=None, dtype=None, copy=False):
 
     @property
     def _constructor(self):
+        """Used when a manipulation result has the same dimesions as the
+        original.
```

Comment:
more like: 

construct a class of this type with the same dimensions as the original